Source/Core/Core/Config/SessionSettings.h

```diff
@@ -10,4 +10,8 @@
 namespace Config
 {
 extern const Info<bool> SESSION_USE_FMA;
+extern const Info<bool> SESSION_LOAD_IPL_DUMP;
+extern const Info<bool> SESSION_GCI_FOLDER_CURRENT_GAME_ONLY;
```

Comment:
This one probably doesn't need "current game only" in the name anymore.

Reply:
I think you've misunderstood its purpose.

Reply:
Yes, I have.